packages/@tinacms/schema-tools/src/types.ts

```diff
@@ -483,6 +484,7 @@ export interface Template {
 export interface FieldCollection {
   label?: string
   name: string
+  nameOverride?: string
```

Comment:
@jamespohalloran this type is for the collection definition, is this needed here?

Reply:
I removed it when I merged with the schema types update so can add back if necessary

Reply:
Oops yeah looks like it should just to be on the field: https://github.com/tinacms/tinacms/pull/3627/files#diff-88b24591b0c36f508270e9ae261e94ea2770198c387d66a95b2fe1620810bde3R147